src/runtime_src/core/common/api/xrt_kernel.cpp

```diff
@@ -1086,7 +1086,7 @@ class argument
     size_t size;   // size in bytes of argument per xclbin
 
     explicit
-    global_type(size_t bytes)
+    global_type(size_t bytes = 0)
```

Comment:
Is this change even used?

Reply:
actually for global_type size is not used but it was present before my changes so made it use default arg.
May be I will remove the arg completely as its not used.

Reply:
I have removed the size arg completely as it is not used in any member function of the class

Reply:
Huh?  The size argument is used at line 1185.   Can you just leave this global_type ctor unchanged?

Reply:
sure Soren, I will add it back.
But the size is used only for construction in 1185 but not used in the class constructed is what I was telling.